index.html

```diff
@@ -4,9 +4,11 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-    <link rel="stylesheet" href="./CSS/styles.css/"">
+    <link rel="stylesheet" href="./CSS/styles.css" />
```

Comment:
You have 2 CSS Links

Reply:
Soweit ich das verstehe ist das eine die alte und das andere die upgedatete Version, wiel ich die noch nicht gemerged habe.